Documentation/kubernetes.md

```diff
@@ -21,28 +21,41 @@ You will need to use the yaml files from the [1.5 folder](/cluster/examples/kube
 
 ### Prerequisites
 
-To make sure you have a Kubernetes cluster that is ready for `Rook`, you can [follow these quick instructions](k8s-pre-reqs.md), including:
-- The `kubelet` requires access to `modprobe` and `rbd` on host
+To make sure you have a Kubernetes cluster that is ready for `Rook`, you can [follow these quick instructions](k8s-pre-reqs.md).
 
 Note that we are striving for even more smooth integration with Kubernetes in the future such that `Rook` will work out of the box with any Kubernetes cluster.
 
 If you are using `dataDirHostPath` to persist rook data on kubernetes hosts, make sure your host has at least 5GB of space available on the specified path.
 
 ### Deploy Rook
 
-With your Kubernetes cluster running, Rook can be setup and deployed by simply creating the rook-operator deployment and creating a rook cluster. To customize the operator settings, see the [Operator Helm Chart](helm-operator.md).
+With your Kubernetes cluster running, Rook can be setup and deployed by simply creating the rook-operator deployment and creating a rook cluster. The rook-operator will automatically deploy a Rook-agent on all the nodes. Rook-agents perform operations to consume storage from the cluster. Refer to the [Rook-agent design document](/design/local-node-agent.md) for more information.
```

Comment:
the rook agent part here sounds like an implementation detail and shouldn't be mentioned in the basic steps here for the user to get Rook running.

Reply:
ok.. ill remove it